tests/utils/test_log_handlers.py

```diff
@@ -203,6 +203,66 @@ def task_callable(ti):
         # Remove the generated tmp log file.
         os.remove(log_filename)
 
+    @pytest.mark.parametrize(
+        "executor_name",
+        [
```

Comment:
I think that It would be interesting to add here _None_ to test the default case.